docs/v3-to-v4.md

```diff
@@ -4,6 +4,7 @@ Version 4 of Node Redis is a major refactor. While we have tried to maintain bac
 
 ## All of the Breaking Changes
 
+If using typescript set the target to ES2015 (ES6) or higher, ES5 and bellow are not supported due to the usage of private class functions.
 See the [Change Log](../CHANGELOG.md).
```

Comment:
Is there also a specific Typescript version (or range) we should expect this change?
```suggestion
If using Typescript set the target to ES2015 (ES6) or higher, ES5 and below are not supported due to the usage of `private` class functions.
See the [Change Log](../CHANGELOG.md).
```

Reply:
It works on Typescript v4 and should work on v3. Not sure about other versions but i would always use the most recent version if possible.
And thnx for the correction